common/lib/xmodule/xmodule/error_module.py

```diff
@@ -30,62 +37,56 @@
 
 class ErrorFields(object):
     """
-    XBlock fields used by the ErrorModules
+    XBlock fields used by the ErrorBlocks
```

Comment:
Could you also update this comment: https://github.com/edx/edx-platform/blob/e9853a9d1db908062e0af78f8524815d4edcb0bc/common/lib/xmodule/xmodule/error_module.py#L32

Also, it looks like `ErrorModule` still mentioned in some comments in the codebase.